style.css

```diff
@@ -0,0 +1,83 @@
+* {
+  font-family: sans-serif;
+  padding-left: 1%;
+}
+
+body {
+  width: 375px;
+  height: 718px;
+}
+
+.menu_line {
+  width: 35px;
+  height: 5px;
+  background-color: #6070ff;
+  margin: 6px 0;
+}
+
+.logo {
+  color: #6070ff;
+  font-style: italic;
+  font-weight: 700;
+  font-size: 20px;
+  text-decoration: none;
+}
+
+.introName {
+  margin-top: 3rem;
+  height: 146px;
+  width: 80%;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  text-align: left;
+}
+
+.intro {
+  width: 327px;
+  height: 150px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  text-align: left;
+}
+
+.connect {
+  color: #7f8cff;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.topHeader {
+  width: 375px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.media_links {
+  display: flex;
+  justify-content: flex-start;
+  gap: 20px;
+  list-style: none;
+  margin-top: -1rem;
+  font-size: 24px;
+}
+
+.backgroundImg {
+  background-image: url(img/header-shapes\ mobile.png);
```

Comment:
:red_circle: I think this background image you are trying to display doesn't exist. Kindly make sure it is part of the project files.